__tests__/app.test.js

```diff
@@ -25,3 +25,22 @@ describe("GET /api", () => {
       });
   });
 });
+
+
+describe("GET /api/topics", () => {
+  test("gives status of 200 and responds with an object of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { allTopics } }) => {
+        allTopics.forEach((topic) => {
```

Comment:
You must have a length check before using a forEach. If the array is empty this loop is skipped and all your expect statements are inside it.